cobalt/docker/linux/Dockerfile

```diff
@@ -34,3 +34,4 @@ ENV SCCACHE_GCS_BUCKET=githubactions-chrome-sccache
 ENV SCCACHE_GCS_RW_MODE=READ_WRITE
 ENV SCCACHE_GCS_KEY_PATH=/root/key.json
 ENV SCCACHE_GCS_SERVICE_ACCOUNT=github-actions-bucket-access@cobalt-demo-330821.iam.gserviceaccount.com
+ENV IS_CI=1
```

Comment:
Technically this line should belong in `.github/` yaml code somewhere, that passes it to container via `-e` environment variable flag. "I am in a container" is not synonymous with "i am running in CI"

But given `sccache` config lives here for no, we can leave it - perhaps leave a todo to factor both out later.

Reply:
Done, added TODO bug